kview/src/main/java/dev/ikm/komet/kview/mvvm/viewmodel/StampCreateFormViewModel.java

```diff
@@ -87,11 +89,31 @@ public void resetOrClearForm(ActionEvent actionEvent) {
     @Override
     public void submitOrConfirm() {
         save();
-
+        showSucessToast();
+        setPropertyValue(IS_CONFIRMED_OR_SUBMITTED, true);
         EvtBusFactory.getDefaultEvtBus().publish(topic, new ClosePropertiesPanelEvent("Stamp From View Model cancel()",
                 ClosePropertiesPanelEvent.CLOSE_PROPERTIES));
     }
 
+    private void showSucessToast() {
+        State status = getValue(STATUS);
+        EntityFacade module = getValue(MODULE);
+        EntityFacade path = getValue(PATH);
+
+        String statusString = getViewProperties().calculator().getPreferredDescriptionTextWithFallbackOrNid(status.nid());
+        String moduleString = getViewProperties().calculator().getPreferredDescriptionTextWithFallbackOrNid(module.nid());
+        String pathString = getViewProperties().calculator().getPreferredDescriptionTextWithFallbackOrNid(path.nid());
```

Comment:
Can we check with @swaroopsalvi about using `.getPrerredDescriptionTextWithFallbackOrNid()`?
I think it is fine, but I thought there was a bug in the method and will always return a regular name. Let's ask @rbradley813  or @aks8m if they can weigh-in too.

Reply:
@carldea @swaroopsalvi  talked with @swaroopsalvi and changed the call to `getDescriptionTextOrNid`
Thanks for the heads up.